Realm/RealmSwift/Migration.swift

```diff
@@ -57,3 +58,42 @@ public class Migration {
         self.rlmMigration = rlmMigration
     }
 }
+
+public class MigrationObject : Object {
+    subscript(key: String) -> AnyObject? {
+        if (self.objectSchema[key].type == RLMPropertyType.Array) {
+            return listProperties[key]
+        }
+        return super[key]
+    }
+
+    public func initalizeListPropertyWithName(name: String, rlmArray: RLMArray) {
+        var list = List<Object>()
```

Comment:
Couldn't these 3 lines could be replaced with `listProperties[name] = List<Object>(rlmArray)`?